configurator/src/main/java/com/devonfw/ide/configurator/logging/Log.java

```diff
@@ -21,11 +25,19 @@
   static {
     try {
       FileHandler logFileHandler = new FileHandler("Configurator.log");
-      logFileHandler.setLevel(Level.ALL);
```

Comment:
java.util.logging is kind of strange. Even though it looks like every level will be logged with this config, nothing below INFO was logged. You also need to tweak the root logger as I do in my fix below.
Also I found the multiline log format very ugly and hard to read so I created my own and use it both for console and for logfile.